gravitee-am-service/src/main/java/io/gravitee/am/service/model/NewDictionary.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.am.service.model;
+
+/**
+ * @author Eric LELEU (eric.leleu at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+public class NewDictionary {
```

Comment:
NewI18NDictionary ?

Reply:
Do we really need separate pojos for create and update?

Reply:
For the update probably not, but then remember to controle the consistency of the ID attribute with the value provided in the url as path parameter.

For the NewDictionary, I don't now. Maybe we should in order to restrict parameters provided by the client because only the language, the name and the default flag is useful for creation. 

Agree to say that we have a lot of POJOs but IMHO having a split between the Objects exposed at the API level and the one use inside the service is useful (maybe a technical task to open in order to refactor all of these Pojos)

Reply:
Yes, agreed.